src/gui/guiFormSpecMenu.cpp

```diff
@@ -2686,6 +2687,83 @@ void GUIFormSpecMenu::parseSetFocus(const std::string &element)
 		<< "'" << std::endl;
 }
 
+void GUIFormSpecMenu::parseMesh(parserData *data, const std::string &element)
+{
+	std::vector<std::string> parts = split(element, ';');
+
+	if (parts.size() < 4 || (parts.size() > 7 &&
+			m_formspec_version <= FORMSPEC_API_VERSION)) {
+		errorstream << "Invalid mesh element (" << parts.size() << "): '" << element
+			<< "'" << std::endl;
+		return;
+	}
+
+	parts.resize(std::max((s32)parts.size(), 7));
+
+	std::vector<std::string> v_pos = split(parts[0], ',');
+	std::vector<std::string> v_geom = split(parts[1], ',');
+	std::string meshstr = unescape_string(parts[2]);
+	std::vector<std::string> textures = split(parts[3], ',');
+	std::vector<std::string> vec_rot = split(parts[4], ',');
+	bool inf_rotation = is_yes(parts[5]);
+	bool mousectrl = !is_yes(parts[6]); // default true
```

Comment:
This does not only make it true by default. (Eg. in the devtest test formspec, it's possible to rotate the chest with the mouse.)
Suggestion for fix:
```suggestion
	bool mousectrl = is_yes(parts[6]) || parts[6].empty(); // default true
```